starting-oss-projects.md

```diff
@@ -169,12 +169,36 @@ Company-based vs. foundation-driven, when to select which approach, leveraging a
 
 templates, health files, license, readme, contribution guidelines
 
+#### Tooling
+
+Creation of repositories can be supported by repository creation tools. These tools create repositories based on some input data via the API's of the Git provider (GitHub.com, GitLab.com, Bitbucket.org etc.). Thus, they can help that repositories are compliant to the company guidelines (contain the required files and team structure, for example). Based on such tools self services for repository creation could be offered that allow development teams creating repositories themselves.
```

Comment:
Maybe it would be good to name some examples or reference a list of such tools at another location.

Maybe it also would be worth stressing user management. I would think that usually not the creation of the repository itself is the main problem but managing who has access to it and for example making sure that people leaving a company get their write privileges removed, etc.

Reply:
Parts of this can be covered by the [opensource-portal](https://github.com/microsoft/opensource-portal) (at least for GitHub.com). I would like to cover that in a separate section.